### DISCUSSION TITLE
fps op doesn't seem to output any fps

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#470] [! Welcome - readme first!]
in chrome m1 macbookpro

**Comment by P-Ahola** [comment]
Same on windows 10 desktop on Firefox.

**Reply by marioepsley** [reply]
just a heads up if you need it, but found out from the meetup that this is how you build a fps counter, also there's and op called triggers per second which does the same thing.

<img width="360" alt="Screenshot 2023-02-02 at 10 57 10" src="https://user-images.githubusercontent.com/36418408/216306782-bac31976-cbc0-4644-b6c8-f20b28b07c1a.png">

**Comment by pandrr** [comment]
fixed on dev.cables.gl
